docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,35 @@
+List Accounts
+=============
+
+This API returns all local user accounts.
+
+The api is::
+
+    GET /_synapse/admin/v1/users?access_token=<admin_access_token>&start=0&limit=10
+
+including an ``access_token`` of a server admin.
+The parameters ``start`` and ``limit`` are optional and are used for pagination.
```

Comment:
convention among matrix pagination APIs (which I think we should copy here) is that pagination apis return a `next_token` value, which you then use for the next iteration. Also, the parameter for the starting point is normally called `from` rather than `start`.

Reply:
This API uses offset paging instead of token based paging. See also https://www.moesif.com/blog/technical/api-design/REST-API-Design-Filtering-Sorting-and-Pagination/# for the limitation, but for a reasonable amount of users this should be OK.

Reply:
Still my point remains: let's not use a different pagination system to the other matrix APIs.

I don't mind if the value that you return from `next_token` is the offset, so that it can be used in the same way.